### PR TITLE
[CFM-1206] CTA block for group actions.

### PR DESCRIPTION
**Pull request by ybaras**
[CFM-1206](https://issuetracker.amplexor.com/jira/browse/CFM-1206)

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module [hunk 1]
```diff
@@ -592,12 +593,17 @@ function c4m_user_og_c4m_visitor_cta($group) {
     ),
   );
 
+  $login_url = t('Please <a href="@login-page">log in</a> first if you have a C4D account', array(
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: We never add markup inside a `t()` function.

please use the `l()` function to create a link.

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module [hunk 2]
```diff
@@ -714,6 +714,8 @@ function c4m_user_og_group_management_cta($group) {
       '#icon' => 'fa-cogs',
       '#text' => l(t("Manage !group_type", array('!group_type' => $group->type)), 'manage'),
     );
+  } elseif (og_is_member('node', $group->nid)) {
+    $items['#title'] = t("You're a member of this !group_type", array('!group_type' => $group->type));
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module]: @ybaras 
Please use single quotes.

**Comment by ybaras** [comment]
imgonline-com-ua-twotoone-w7mflu7w1nke

**Comment by ordavidil** [comment]
closing in favor of #1515

**Pull request closed without merging**